autoscheduler/scraper/models/section.py

```diff
@@ -19,7 +18,7 @@ class Section(models.Model):
 
     max_enrollment = models.IntegerField()
     current_enrollment = models.IntegerField()
-    instructor = models.ForeignKey(Instructor, on_delete=models.CASCADE)
+    instructor = models.ForeignKey('Instructor', on_delete=models.CASCADE)

```

Comment:
So the `from scraper.models import Instructor` was giving me a `cyclic-import` error only in Actions(not locally), but we shouldn't even have to import Instructor anyways as we can just specify the name of the model instead.